src/commands/app/list-upgrade-candidates.ts

```diff
@@ -0,0 +1,68 @@
+import { Simplify } from "@mittwald/api-client-commons";
+import {
+  appInstallationArgs,
+  withAppInstallationId,
+} from "../../lib/app/flags.js";
+import { ListBaseCommand } from "../../ListBaseCommand.js";
+import { MittwaldAPIV2, MittwaldAPIV2Client } from "@mittwald/api-client";
+import { ListColumns } from "../../Formatter.js";
+import { getAppInstallationFromUuid } from "../../lib/app/uuid.js";
+
+type ResponseItem = Simplify<
+  MittwaldAPIV2.Paths.V2AppsAppIdVersions.Get.Responses.$200.Content.ApplicationJson[number]
+>;
+
+type Response = Awaited<
+  ReturnType<MittwaldAPIV2Client["app"]["listAppversions"]>
+>;
+
+export default class List extends ListBaseCommand<
+  typeof List,
+  ResponseItem,
+  Response
+> {
+  static description = "List update candidates for an appInstallation.";
```

Comment:
Wording consistency: "upgrade" vs. "update"

```suggestion
  static description = "List upgrade candidates for an app installation.";
```